src/librustdoc/passes/collect_trait_impls.rs

```diff
@@ -64,6 +65,7 @@ crate fn collect_trait_impls(mut krate: Crate, cx: &mut DocContext<'_>) -> Crate
         map: &FxHashMap<DefId, &Type>,
         cleaner: &mut BadImplStripper,
         type_did: DefId,
+        c: &Cache,
```

Comment:
`cx` is already passed in the parameters, so no need to add the `cache` in here. Just call `cx.cache`.